search/lib/settings.js

```diff
@@ -52,6 +52,7 @@ function getSettings () {
     // comma separated list of URL aliases
     settings.cmrStacRouteAliases = process.env.CMR_STAC_ROOT_ALIASES || '/cmr-stac';
     settings.cmrStacRelativeRootUrl = process.env.CMR_STAC_RELATIVE_ROOT_URL || '/stac';
+    settings.cmrCloudStacRelativeRootUrl = process.env.CMR_CLOUDSTAC_RELATIVE_ROOT_URL || '/cloudstac';
```

Comment:
Recommend deleting this and using CMR_STAC_RELATIVE_ROOT_URL as the root still. Set behavior with something like "IS_CLOUD_ONLY".

Reply:
We were talking about that and decided to go with a new route. It's a big change to go back to /stac/IS_CLOUD_ONLY.